modules/ROOT/pages/configure-service-uri-task.adoc

```diff
@@ -0,0 +1,19 @@
+= Configuring the Service URI
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+When using a load balancer in front of a RAML or OAS API implementation, you can enable the *Keep RAML/OAS base URI* flag. This ensures that the URL specified in the proxy matches the one in your API.
```

Comment:
Is "in front of" correct? Is that how users think about it?

Reply:
According to ChatGPT and some Google searches, yes.
"In front of" is the proposition that best collocates with "load balancer".
Thanks!